config/settings.yaml.example

```diff
@@ -22,6 +22,10 @@
 # errors in your browser
 :webpack_dev_server_https: false
 
+# If you set webpack_dev_server_https you most likely also want to specify hostname
+# matching your SSL certificate
+:webpack_dev_server_host: foreman.example.com
```

Comment:
Given people might just copy-paste the example file, I think we should keep this setting commented out: otherwise it would not work unless the user actually has it's host set to `foreman.example.com`